katdal/chunkstore.py

```diff
@@ -38,6 +38,11 @@ class StoreUnavailable(OSError, ChunkStoreError):
     """Could not access underlying storage medium (offline, auth failed, etc)."""
 
 
+class NotSupported(ChunkStoreError):
```

Comment:
This name is a bit vague... How about `UnsupportedStoreFeature`?

Reply:
Fair enough - changed